Projects/My_website/myWebsite/blog/views.py

```diff
@@ -17,8 +18,9 @@ def posts(request):
 
 def postDetail(request,slug):
     iPost = Post.objects.all().get(slug = slug)
```

Comment:
@codal-uchoudhary , are you sure this is the way we should be fetching an individual post instance?